app/views/dashboard/_pending_applications.haml

```diff
@@ -10,5 +10,9 @@
               %li= link_to record.user.proper_name, record
           - else
             No pending applications
-        = link_to 'View saved applications for ' + position.name,
-          saved_applications_position_path(position)
+        - if @saved_records[position].present?
+          - count = @saved_records[position].count
+          %ul= link_to "View saved applications for #{position.name} (#{count})",
```

Comment:
Since both cases include a `ul`, it'd be better to write it outside of the conditional.